apps/kyc-server/src/components/dashboard/table/user-table-filters.tsx

```
@@ -0,0 +1,188 @@
+	SelectTrigger,
+	SelectValue,
+} from '~/components/base/select'
+
+interface UserTableFiltersProps {
+	search: string
+	statusFilter: string
+	verificationLevelFilter: string
+	onSearchChange: (search: string) => void
+	onStatusFilterChange: (status: string) => void
+	onVerificationLevelFilterChange: (level: string) => void
+	table: Table<any>
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)